src/api/authorityApi.ts

```diff
@@ -12,14 +12,16 @@ export enum AuthorityApiPaths {
 }
 
 export const getAuthorities = async (name: string, dispatch: Dispatch) => {
+  const url = encodeURI(`/authority?name=${name}`);
```

Comment:
sjekket at denne virker? Jeg prøvde samme selv for å hente prosjekt uten suksess. måtte bruke `/authority?name=${encodeURIComponent(name)}`. prøv med komma feks

Reply:
den fungerer ja. mulig BARE-proxyen gjør noe fancy i backend som Cristin ikke gjør da